projects/js-packages/publicize-components/src/components/message-box-control/index.tsx

```diff
@@ -3,25 +3,54 @@ import { TextareaControl } from '@wordpress/components';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { useCallback, useRef } from 'react';
 
+export const PLACEHOLDER_TEXT = __(
+	'Write a custom message for your social audience here. This message will override your social post content.',
+	'jetpack-publicize-components'
+);
+export const DEFAULT_LABEL = __( 'Message', 'jetpack-publicize-components' );
+
+export type MessageBoxControlProps = {
+	/** The label for the message box */
+	label?: string;
```

Comment:
`label` should be required for a11y or we should allow passing of `aria-label`.

Reply:
Done 32a9397